src/styles/blocks/card.scss

```
@@ -0,0 +1,92 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    display: block;
+  }
+}
+
+.card__info {
+  margin-top: 40px;
```

Use scss nesting, fix it everywhere
```suggestion
  }
  
  &__info {
    margin-top: 40px;
  }
}

```